libs/Wallet/qml/Status/Wallet/AssetsPanel.qml

```diff
@@ -109,32 +123,72 @@ Item {
             onClicked: newAccountLoader.active = true
         }
 
-        Loader {
-            id: newAccountLoader
+        ColumnLayout {
+            visible: !errorLayout.visible && newAccountLoader.active
 
-            Layout.fillWidth: true
+            Rectangle {
+                color: "blue"
+
+                Layout.fillWidth: true
+                Layout.preferredHeight: 2
+                Layout.margins: 5
+            }
 
-            visible: !errorLayout.visible && active
-            active: false
+            ComboBox {
+                id: accountTypeComboBox
 
-            sourceComponent: Component {
-                NewWalletAccountView {
-                    controller: root.controller.createNewWalletAccountController()
+                textRole: "title"
+                valueRole: "sourceComponent"
 
-                    onCancel: newAccountLoader.active = false
-                    onAccountCreated: newAccountLoader.active = false
+                Layout.fillWidth: true
 
-                    Connections {
-                        target: controller
-                        function onAccountCreatedStatus(createdSuccessfully) {
-                            if(createdSuccessfully)
-                                newAccountLoader.active = false
-                            else
-                                errorLayout.visible = true
+                component NewAccountEntry: ItemDelegate {
+                    required property string title
+                    required property Component sourceComponent
+
+                    text: title
+                    width: accountTypeComboBox.width
+                }
+                model: ObjectModel {
+                    NewAccountEntry {
+                        title: "New Account"
```

Comment:
Not totally necessary at this point, but `qsTr()`. These "temp" things tend to get forgotten in the end :)